src/app/components/pitch/pitch.component.ts

```
@@ -0,0 +1,270 @@
+@Component({
+  selector: 'app-pitch',
+  imports: [FormsModule],
+  templateUrl: './pitch.component.html',
+  styleUrls: ['./pitch.component.scss']
+})
+export class PitchComponent implements AfterViewInit {
+  @ViewChild('footballCanvas', { static: false }) canvasRef!: ElementRef<HTMLCanvasElement>;
+  @ViewChild('timebar', { static: false }) timebarRef!: ElementRef<HTMLInputElement>;
+
+  private ctx!: CanvasRenderingContext2D | null;
+  public currentTime: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)